modules/ssm-parameters/README.md

```diff
@@ -0,0 +1,100 @@
+# Component: `ss-parameters`
+
+This component is responsible for provisioning Parameter Store resources against AWS SSM. It supports normal parameter store resources that can be configured directly in YAML OR pulling secret values from a local Sops file.
```

Comment:
Could this component be used for both SSM params and AWS Secrets Manager ? or would it be best to have a separate component for the latter ?

Reply:
Ah I could definitely see it being upgraded for Secrets Manager usage as well. We typically don't use Secrets Manager as there aren't that many use-cases IMO, but I could definitely see the need for folks that use it! I like the idea... 

In that case, we could rename the component? Not sure how far you want to take this idea 😅